raster-foundry/raster-foundry
# Add single band parameters to project model and api

## Overview

Add single band options to project model and api

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~

## Testing Instructions
* Run migrations
* Verify that tests complete
* Verify that the api matches the swagger spec
* Verify that you can set the new project properties
* Verify that existing projects works as expected (IE I didn't break the frontend by making incompatible api changes)

Closes #2323
